uber/site_sections/preregistration.py

```diff
@@ -260,7 +260,7 @@ def dealer_confirmation(self, session, id):
     @log_pageview
     def group_members(self, session, id, message=''):
         group = session.group(id)
-        charge = Charge([group, group.leader]) if group.leader else Charge(group)
+        charge = Charge(group)
```

Comment:
This is a page where we explicitly _don't_ want to charge for the attendees' badges. Groups have their own properties/columns for money.